homework.rb

```diff
@@ -1,6 +1,20 @@
-VARIANT = 0
+VARIANT = 2
 
 def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+	"#{digits}-#{letters}-#{digits}"
 end
+
+def digits
+	rand(1000..9999).to_s
+end
+
+def letters
+	letters, letter, i = ("A".."Z").to_a, "", 0
```

Comment:
"letter" is incorrect name, because you try to add 4 letters into variable. Name of variables, methods and classes should explain context, type, sense and other